### DISCUSSION TITLE
Tutorial > Third-Party JavaScript snippet page not working?

### DISCUSSION THREAD

**Discussion by ogre-yoga** [vercel/next.js#33816] [Help] [answered]
Hello all!

I'm following through the tutorial and the demo on [Third Party Javascript](https://nextjs.org/learn/basics/assets-metadata-css/third-party-javascript) is not working for me:

- window.FB is undefined
- no console.log statement in Web Inspector

I've tried with other external scripts and they don't seem to be loading either, but I am not getting any warnings or errors so I'm not sure how to troubleshoot. Any guidance would be much appreciated.

Ref: /pages/post/first-post

```
import Layout from '../../components/layout'
import Head from 'next/head'
import Script from 'next/script'
import Link from 'next/link'

export default function FirstPost() {
  return (
    <Layout>
      <Head>
        <title>First Post here!</title>
        <Script
          src="https://connect.facebook.net/en_US/sdk.js"
          strategy="lazyOnload"
          onLoad={() =>
            console.log(`script loaded correctly, window.FB has been populated`)
          }
        />
      </Head>
      <h1>First Post</h1>
      <h2>
        <Link href="/">
          <a>Back to home</a>
        </Link>
      </h2>
    </Layout>
  )
}
```

<img width="1345" alt="Screen Shot 2022-01-31 at 8 31 50 am" src="https://user-images.githubusercontent.com/49114586/151718779-f8d394fa-ae0b-4f4b-bbdd-a5ae46980b8e.png">

**Comment by wwwdepot** [comment, marked as answer]
Tutorial is using `Script` after `Head`. Please move the Script component and try again. 

```jsx
      <Head>
        <title>First Post</title>
      </Head>
      <Script
        src="https://connect.facebook.net/en_US/sdk.js"
        strategy="lazyOnload"
        onLoad={() =>
          console.log(`script loaded correctly, window.FB has been populated`)
        }
      />
```

**Reply by ogre-yoga** [reply]
Argh! Obviously decades of HTML nesting habits die hard. Thank you for the second pair of eyes.

**Reply by songyang-dev** [reply]
I've been stuck on this too. Thanks for the answer!
